tests/test_escaping.py

```diff
@@ -1,6 +1,13 @@
 import warnings
 from bs4 import MarkupResemblesLocatorWarning
-from markdownify import markdownify as md
+from markdownify import MarkdownConverter
+
+
+def md(html, **options):
+    # disable document-level stripping so separation newlines are included in testing
+    options = {**options, "strip_document": None}
+
+    return MarkdownConverter(**options).convert(html)
```

Comment:
Could we refactor this into some sort of utils file in the tests, then import it from there? This would remove the duplicate code in every test file.

Reply:
Good idea! I will have a look at that.

Reply:
Done! Good idea.